lib/pages/onBoarding_page.dart

```diff
@@ -21,30 +29,44 @@ class _OnBoardingPageState extends State<OnBoardingPage> {
             child: Column(
```

Comment:
make its `MainAxisAlignment` to `center`

Reply:
Done :+1: